compiler/transforms/convert_acc_to_csr.py

```diff
@@ -41,7 +45,9 @@ def get_acc(self, accelerator: StringAttr) -> acc.AcceleratorOp:
             raise RuntimeError(
                 f"Invalid IR: no accelerator op for @{accelerator.data} found in module"
             )
-        return acc_op
+        # Get accelerator interface with symbolref without @
+        acc_info = get_registered_accelerators()[str(acc_op.name_prop)[1:]]
```

Comment:
can we emit a more informative error if the accelerator in the module is not registered?

Reply:
Adapted!